dist/manifest.json

```diff
@@ -11,18 +11,5 @@
   "background": {
     "service_worker": "background.js"
   },
-  "host_permissions": [
-    "https://scholar.google.com/*",
-    "https://arxiv.org/*",
-    "https://aclanthology.org/*",
-    "https://proceedings.mlr.press/*",
-    "https://www.jmlr.org/*",
-    "https://proceedings.neurips.cc/*",
-    "https://proceedings.nips.cc/*",
-    "https://openreview.net/*",
-    "https://openaccess.thecvf.com/*",
-    "https://ieeexplore.ieee.org/*",
-    "https://www.biorxiv.org/*",
-    "https://dl.acm.org/*"
-  ]
+  "host_permissions": ["https://*/*"]
```

Comment:
I'm not really sure about the performance hit of running this on _every single site_… it appears that it's not possible to match only specific paths on each host (e.g. `/*/doi/*` or `/*/pdf/*`).